lib/paths.cf

```diff
@@ -650,6 +650,10 @@ bundle common paths
       "tmp_path[$(tmp_paths)]" string => "$(files_path)$(path[$(tmp_paths)])";
       "path[$(tmp_paths)]" string => "$(tmp_path[$(tmp_paths)])";
 
+    windows:
+      "path[true]"          string => "exit 0"; # designed to be used with contain => in_shell
```

Comment:
Hummmm, I dunno about htis one. what if it was used with powershell instead? also, i don't think that it will get the _stdlib_path_exists_true class since there is no file at path "exit 0". and the dummy true commands only run if that is defined.